### PR TITLE
[BLCKCHN-202] support google gemini #25

### PR DESCRIPTION
**Pull request by leejw51crypto**
- add `gemini` to ai service
- add `gemini` to python chat

### CODE REVIEW

File: ai/cryptocom-ai-agent-pychatbot/chat.py [hunk 1]
```diff
@@ -7,33 +7,53 @@
...
+# Check both API keys after load_dotenv()
 api_key = os.getenv("OPENAI_API_KEY")
+google_api_key = os.getenv("GOOGLE_API_KEY")
+
 if not api_key:
     print("Error: OPENAI_API_KEY not found in environment variables.")
     print("Please make sure you have set the OPENAI_API_KEY in your .env file.")
     exit(1)
 else:
     print("✓ OPENAI_API_KEY is successfully ready")
```
Comment (rarcifa) [inline ai/cryptocom-ai-agent-pychatbot/chat.py:19]: has been imported successfully

File: ai/cryptocom-ai-agent-pychatbot/chat.py [hunk 2]
```diff
@@ -7,33 +7,53 @@
...
-    print()
 
+if not google_api_key:
+    print("Error: GOOGLE_API_KEY not found in environment variables.")
+    print("Please make sure you have set the GOOGLE_API_KEY in your .env file.")
+    exit(1)
+else:
+    print("✓ GOOGLE_API_KEY is successfully ready")
```
Comment (rarcifa) [inline ai/cryptocom-ai-agent-pychatbot/chat.py:26]: has been imported successfully

File: ai/cryptocom-ai-agent-pychatbot/chat.py [hunk 3]
```diff
@@ -7,33 +7,53 @@
...
-def send_query(query: str, context: list = None) -> dict:
+
+def send_query(query: str, context: list = None, provider: str = "gemini") -> dict:
     """
     Send a query to the AI Agent Service and return the response.
 
     Args:
         query (str): The natural language query to send
+        context (list, optional): Context for the conversation
+        provider (str, optional): LLM provider to use ("openai" or "gemini")
```
Comment (rarcifa) [inline ai/cryptocom-ai-agent-pychatbot/chat.py:38]: provider = "gemini"

(str, optional): LLM provider to use (provider)

**Review by yeehan-crypto-com — APPROVED** [review]
LGTM, added some commits to add comments in the code.

**Comment by leejw51crypto** [comment]
ok

**Pull request merged**